django_ses/__init__.py

```diff
@@ -141,7 +141,7 @@ def send_messages(self, email_messages):
 
                 if len(message.to) + len(message.cc) + len(message.bcc) == 0:
                     logger.debug('Refusing to send email. All recipients were filtered by the blacklist')
-                    return
+                    continue
```

Comment:
This makes sense to me.

As the original user for this feature @alexandernst what do you think?